java/src/main/java/com/ncc/aif/AIFUtils.java

```diff
@@ -995,6 +995,52 @@ public static Resource markAudioJustification(Model model, Collection<Resource>
         return justification;
     }
 
+    /**
```

Comment:
@ecurley-nextcentury ,

Do we need a markVideoJustification also?

Reply:
The mark...() APIs are difficult to maintain and don't save anyone much. All the parameters for make...() are still required and it saves a single call to markJustification. Let me know if that makes sense to you. I'm happy to hear counterpoints.